### ISSUE TITLE
Not works with repository as chart

### ISSUE THREAD

**Issue by aight8** [databus23/helm-diff#9]
```
helm diff gitlab gitlab/gitlab --values values/gitlab.yaml
Error: file "gitlab/gitlab" not found
```

**Comment by njam** [comment]
@databus23 @elemental-lf could it be that this is solved by #20?

**Comment by elemental-lf** [comment]
@njam it's hard to say as the error message was very generic but I'd guess that your assumption is correct. So this could probably be closed.
@aight8 are you still using helm-diff? If so, could you verify if your problem persists with the current release?

**Comment by databus23** [comment]
As stated above I believe this is resolved by #20
